source/_posts/2015-10-17-headless-firefox-in-ubuntu-on-virtual-box-for-cucumber-testing.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Headless Firefox in Ubuntu on VirtualBox for Cucumber testing"
+date: 2015-10-17 14:12:56 +0200
+comments: true
+author: Ivica Lakatoš
+categories: 
+  - Ruby and Rails
+  - Testing
+tags: 
+  - ruby
+  - rails
+  - vagrant
+  - virtualbox
+  - ubuntu
+  - cucumber
+  - selenium-webdriver
+published: true
+---
+
+If you use [Vagrant](http://www.vagrantup.com/downloads.html), [VirtualBox](https://www.virtualbox.org/) and Ubuntu to build your Rails apps and you want to test it with Cucumber scenarios, this is right post for you. By default Vagrant and VirtualBox use Ubuntu without an X server and GUI.
```

Comment:
@Ikera I think this part "this is right post for you" is missing an article, so it should be "this is the right post for you"